app/controllers/admin/invitations_controller.rb

```diff
@@ -19,12 +19,12 @@ def create
       end
 
       username = "#{name.downcase.tr(' ', '_').gsub(/[^0-9a-z ]/i, '')}_#{rand(1000)}"
+
       User.invite!(email: email,
                    name: name,
                    username: username,
                    remote_profile_image_url: ::Users::ProfileImageGenerator.call,
                    saw_onboarding: false,
-                   editor_version: :v2,
```

Comment:
`editor_version` is no longer called on `User` but on `UsersSetting`, and defaults to 'v2' in `UsersSetting`. so I thought to remove it from this `invite!` action.